packages/bundle/webpack.config.js

```diff
@@ -39,6 +39,9 @@ module.exports = {
     alias: {
       react: resolve(__dirname, 'node_modules/isomorphic-react/dist/react.js'),
       'react-dom': resolve(__dirname, 'node_modules/isomorphic-react-dom/dist/react-dom.js')
-    }
+    },
+    // We are not transpiling using Webpack, thus, we cannot use package.json/module field to load a module.
+    // If we are using "module" field, we could be loading non-ES5 code. So we are forcing Webpack to just use "browser", then "main" field.
```

Comment:
```suggestion
    // If we are using "module" field, we could be loading non-ES5 code. We are forcing Webpack to just use "browser", then "main" field.
```

Reply:
Not a fan of using 'we' in comments. I'll let this one go though :P

Reply:
I use too much. I keep forgetting to change the habit even I proofread it multiple times.

While I wrote research papers in the past, one of my advisors told me, "you should not use I, but we, although you are writing it by yourself."

Reply:
xD